api/types/authentication.go

```diff
@@ -248,6 +254,16 @@ func (c *AuthPreferenceV2) SetAllowLocalAuth(b bool) {
 	c.Spec.AllowLocalAuth = NewBoolOption(b)
 }
 
+// GetMessageOfTheDay gets the current Message Of The Day. May be empty.
+func (c *AuthPreferenceV2) GetMessageOfTheDay() string {
```

Comment:
Not sure if this belongs in `AuthPreferenceV2`? Maybe we should create a new resource? Let's talk about this today.

Reply:
Spoke with @klizhentas - upshot of conversation was that it's probably overkill to break it out into its own resource and this is the least-bad place for it.